FabFos.py

```diff
@@ -247,6 +257,9 @@ def get_options():
                           action="store_true", default=False, required=False)
 
     opts = parser.add_argument_group(title="Optional arguments")
+
+    opts.add_argument("-t", "--type", choices=["B", "F"], required=False, default="F",
+                      help="Enter B if input type is BAM, F for FastQ")
```

Comment:
Always a good idea to leave a note with what the default value is e.g. [ DEFAULT = 'F' ]